src/sql/workbench/contrib/dashboard/browser/core/dashboardTab.contribution.ts

```diff
@@ -282,14 +282,14 @@ const CommonTabs: IDashboardTab[] = [
 		container: {
 			'widgets-container': [
 				{
-					name: localize('databasesWidgetTitle', "Search"),
 					gridItemConfig: {
 						sizex: 3,
-						sizey: 3
+						sizey: 2
 					},
 					widget: {
 						'explorer-widget': {}
-					}
+					},
+					showTitle: false
```

Comment:
Is title necessary for anything? What if you just made title optional and if it wasn't defined then it wouldn't be shown?

Reply:
I am renaming this to showHeader to be more accurate.

this is content of widget header, including icon, title and actions (the '...' button), it is more than just the title, I think it is better to introduce an option for it. 
```
		<div class="widgetHeader">
			<span *ngIf="_config.icon" [ngClass]="['icon', _config.icon]"></span>
			<span *ngIf="_config.name && _showTitle" class="widgetTitle">{{_config.name}}</span>
			<span *ngIf="!_config.name" class="noTitle"></span>
			<span #actionbar class="actionbar"></span>
		</div>
```